basics/variables.md

```diff
@@ -136,6 +165,17 @@ This is not limited to integers.
 >>>
 ```
 
+But, you can only use `+=` and `*=`. Because of with the others operators
+it causes a type error.
+
+```python
+>>> a = 'hello'
+>>> a -= 3
+TypeError: unsupported operand type(s) for -: 'str' and 'str'
+>>>
+```
+
+
```

Comment:
These things are best suited for "Fix this program" exercises IMO. What do you think?